server/models/Profile.js

```diff
@@ -0,0 +1,46 @@
+const mongoose = require("mongoose");
+
+const profileSchema = new mongoose.Schema({
+    userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "User",
+    required: true,
+    },
+    firstName: {
+        type: String,
+        required: true,
+    },
+    lastName: {
+        type: String,
+        required: true,
+    },
+    birthDate: {
+        type: Date,
+        default: '1998-06-15'
+    },
+    email: {

```

Comment:
Please remove the 'email' field as it is already a part of the User model, we wouldn't want to duplicate data within the DB :)

Reply:
Removed!